### PR TITLE
Add k8s manifest file for rayyan-store-proxy

### PR DESCRIPTION
**Pull request by fabubaker**

### CODE REVIEW

File: kubernetes/store-proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        app: rayyan
+        rayyan.role: store-proxy
+      name: rayyan-store-proxy
+    spec:
+      containers:
+      - env:
+        - name: PROXY_PORT
+          value: "3000"
+        - name: REDIS_URL
+          value: rayyan-store-proxy-redis:6379
```
Comment (hammady) [inline kubernetes/store-proxy.yaml]: Don't we need a full URL with scheme, path, ...?

File: kubernetes/store-proxy.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: rayyan
+    rayyan.role: store-proxy
```
Comment (hammady) [inline kubernetes/store-proxy.yaml]:
```
  labels:
    app: rayyan
    rayyan.role: server
    rayyan.task: rayyan-store-proxy
    rayyan.image: rayyan-store-proxy
```

File: kubernetes/store-proxy.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  selector:
+    app: rayyan
+    rayyan.role: store-proxy
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: rayyan
+    rayyan.role: store-proxy-redis
```
Comment (hammady) [inline kubernetes/store-proxy.yaml]:
```
  labels:
    app: rayyan
    rayyan.role: server
    rayyan.task: rayyan-store-proxy-redis
```

File: kubernetes/store-proxy.yaml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  template:
+    metadata:
+      labels:
+        app: rayyan
+        rayyan.role: store-proxy-redis
+      name: rayyan-store-proxy-redis
+    spec:
+      containers:
+      - name: rayyan-store-proxy-redis
+        image: redis
```
Comment (hammady) [inline kubernetes/store-proxy.yaml]: Let's freeze the redis image tag.

File: kubernetes/store-proxy.yaml (hunk 1 above)
Comment (fabubaker) [reply kubernetes/store-proxy.yaml]: From the `go-redis` docs:

```
	client := redis.NewClient(&redis.Options{
		Addr:     "localhost:6379",
		Password: "", // no password set
		DB:       0,  // use default DB
	})
```
So no need for the scheme, at least. Running this on minikube also worked (the proxy was able to `GET` from the redisdb) so this should be fine.

Comment (hammady) [reply kubernetes/store-proxy.yaml]: Is there an option to specify all parameters through a URL? A single configuration is easier to manage, especially when we test on a managed service.
Please check this: https://godoc.org/github.com/go-redis/redis#ParseURL

**Comment by fabubaker** [comment]
https://github.com/rayyanqcri/rayyan-store-proxy/commit/876b2920733348c48892a69d3ba623a4e7d1a162  changes the proxy to use `redis.ParseURL`, so we can now include the full parameters in the URL.

**Pull request merged**